core/service/src/backup/tests.rs

```diff
@@ -17,6 +17,7 @@ use crate::{
     engine::base::derive_request_id,
 };
 use aes_prng::AesRng;
+use alloy_primitives::Address;
```

Comment:
I am wondering if we are still relying too much on arrays in this file, since now hashmaps with verification keys might make more sense in certain places

Reply:
the verification key cannot be the key for the hashmap as it does not implement Ord. I wrote it in a comment here 
https://github.com/zama-ai/kms/pull/296/files#diff-d494ecdbb919122886fc25291a0e4a910989bcd9355e234ad5e0d50c4c379fbcR593

in core/service/src/backup/tests.rs

we could implement Ord I suppose but these hashmaps are only used for managing our tests, so I was hesitant to add something in an important type like PublicSigKey that will only be used for testing